source/agora/consensus/EnrollmentManager.d

```diff
@@ -525,14 +554,26 @@ unittest
     assert(man.addEnrollment(enroll2));
     assert(man.getEnrollmentLength() == 2);
 
+    auto utxo_hash3 = utxo_hashes[2];
+    Enrollment enroll3;
+    assert(man.createEnrollment(utxo_hash3, enroll3));
+    assert(man.addEnrollment(enroll3));
+    assert(man.getEnrollmentLength() == 3);
+
+    Enrollment[] enrolls;
+    man.getUnregisteredEnrollments(enrolls);
+    assert(enrolls.length == 3);
+    assert(enrolls[0].utxo_key < enrolls[1].utxo_key);
```

Comment:
You can just use https://dlang.org/phobos/std_algorithm_sorting.html#isStrictlyMonotonic

Reply:
I knew this.
It was used in #480PR.
I will improve and write a new PR.

Reply:
Oh, Thanks.